src/main/thrift/parquet.thrift

```diff
@@ -197,18 +197,78 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+/** Identifier for the sort order used to produce min and max values. */
```

Comment:
I think sorting columns may need a separate update. These identifiers could potentially be used there, but I don't think the ordering (or direction) is currently stored.